debian/patches/series

```diff
@@ -0,0 +1,3 @@
+fix-make-clean
+add-generated-files
+fix-removal-of-generated-files
```

Comment:
Should probably make the changes needed for `fix-removal-of-generated-files` and `fix-make-clean` into the emulator source tree.